lib/src/models/kyc_requirement.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:kyc_client_dart/kyc_client_dart.dart';
+import 'package:kyc_client_dart/src/api/models/v1_data_type.dart';
+import 'package:kyc_client_dart/src/api/models/v1_document_field_type.dart';
+import 'package:kyc_client_dart/src/api/models/v1_document_type.dart';
+import 'package:kyc_client_dart/src/api/models/v1_formula.dart';
+import 'package:kyc_client_dart/src/api/models/v1_get_kyc_requirements_response.dart';
+
+part 'kyc_requirement.freezed.dart';
+part 'kyc_requirement.g.dart';
+
+enum BasicInfoType {
+  email,
+  phone,
+  selfie,
+}
+
+enum DocumentField {
+  idNumber,
+  photoFront,
+  photoBack,
+  other,
+}
+
+@freezed
+class KycRequirement with _$KycRequirement {
+  const factory KycRequirement({
+    required String country,
+    required List<Requirement> requirements,
+  }) = _KycRequirement;
+
+  factory KycRequirement.fromProto(V1GetKycRequirementsResponse response) {
```

Comment:
These are *requirements*, so any unknown (added in the later versions) entities should not be skipped, but should rather throw an error.